packages/manager/apps/dedicated/client/app/components/dedicated-cloud/taskPollProgress/translations/Messages_fr_FR.json

```
@@ -0,0 +1,17 @@
+{
+  "dedicatedCloud_task_progress_operation": "Opération :",
+  "dedicatedCloud_task_progress_operation_addUser": "Ajout d'un utilisateur",
+  "dedicatedCloud_task_progress_task_reference": "Tâche : {{ task }}",
+  "dedicatedCloud_task_progress_state_done": "Terminé",
```

shouldn't it be 
```suggestion
  "dedicatedCloud_task_progress_state_done": "Terminée",
```
as we are referring to a task ?

---

It's applied